vaas/vaas/cluster/admin.py

```diff
@@ -202,11 +213,21 @@ def labels(self, obj):
         labels_list_html = ''
         if obj.labels:
             for label in obj.labels:
-                labels_list_html += "<div class='span13 text-center'>%s</div>" % label
+                labels_list_html += "<span class='label label-default'>%s</span>" % label
         return format_html(labels_list_html)
 
     labels.short_description = 'Labels'
 
+    def get_domains(self, obj) -> SafeText:
```

Comment:
```
obj: DomainMapping
```

Reply:
You are wrong. Here is LogicalCluster object. But right i focus only here for type the result.